cmd/snap/cmd_snap_op.go

```diff
@@ -635,6 +635,10 @@ func (x *cmdRefresh) Execute([]string) error {
 	}
 
 	if x.List {
+		if len(x.Positional.Snaps) > 0 {
+			return errors.New(i18n.G("--list does not accept positional arguments"))
```

Comment:
I know this is correct, but would _additional_, rather than _positional_, be clearer?